Tests/CodeGenerateTests/CodeGenerateTests.swift

```diff
@@ -1,17 +1,97 @@
+        let swaggerCommand = ["--base-file-path", swaggerFilePath, "--swagger-version", "2"]
+        let openAPICommand = ["--base-file-path", openAPIFilePath, "--swagger-version", "3"]
+        let fileManager = FileManager()
+
+        SmokeFrameworkApplicationGenerateCommand.main(swaggerCommand)
+        SmokeFrameworkApplicationGenerateCommand.main(openAPICommand)
+        
+        let pass = fileManager.contentsEqual(atPath: swaggerFilePath + "/Sources/", andPath: openAPIFilePath + "/Sources/")
+        XCTAssertTrue(pass)
+    }
+    
+    func testPlaybackObjectProcessorServiceGenerate() {
+        let swaggerFilePath = "/Users/mndzup/Desktop/Service_Swagger/PlaybackObjectProcessorService"
+        let openAPIFilePath = "/Users/mndzup/Desktop/Service_OpenAPI/PlaybackObjectProcessorService"
+        let swaggerCommand = ["--base-file-path", swaggerFilePath, "--swagger-version", "2"]
+        let openAPICommand = ["--base-file-path", openAPIFilePath, "--swagger-version", "3"]
+        let fileManager = FileManager()
+
+        SmokeFrameworkApplicationGenerateCommand.main(swaggerCommand)
+        SmokeFrameworkApplicationGenerateCommand.main(openAPICommand)
+        
+        let pass = fileManager.contentsEqual(atPath: swaggerFilePath + "/Sources/", andPath: openAPIFilePath + "/Sources/")
+        XCTAssertTrue(pass)
+    }
+    
+    func testPlaybackObjectsServiceGenerate() {
+        let swaggerFilePath = "/Users/mndzup/Desktop/Service_Swagger/PlaybackObjectsService"
+        let openAPIFilePath = "/Users/mndzup/Desktop/Service_OpenAPI/PlaybackObjectsService"
+        let swaggerCommand = ["--base-file-path", swaggerFilePath, "--swagger-version", "2"]
+        let openAPICommand = ["--base-file-path", openAPIFilePath, "--swagger-version", "3"]
+        let fileManager = FileManager()
+
+        SmokeFrameworkApplicationGenerateCommand.main(swaggerCommand)
+        SmokeFrameworkApplicationGenerateCommand.main(openAPICommand)
+        
+        let pass = fileManager.contentsEqual(atPath: swaggerFilePath + "/Sources/", andPath: openAPIFilePath + "/Sources/")
+        XCTAssertTrue(pass)
     }
+    
+    func testTargetPublishingServiceGenerate() {
+        let swaggerFilePath = "/Users/mndzup/Desktop/Service_Swagger/TargetPublishingService"
+        let openAPIFilePath = "/Users/mndzup/Desktop/Service_OpenAPI/TargetPublishingService"
+        let swaggerCommand = ["--base-file-path", swaggerFilePath, "--swagger-version", "2"]
+        let openAPICommand = ["--base-file-path", openAPIFilePath, "--swagger-version", "3"]
+        let fileManager = FileManager()
 
-    func executionPlanServiceGenerate() {
+        SmokeFrameworkApplicationGenerateCommand.main(swaggerCommand)
+        SmokeFrameworkApplicationGenerateCommand.main(openAPICommand)
         
+        let pass = fileManager.contentsEqual(atPath: swaggerFilePath + "/Sources/", andPath: openAPIFilePath + "/Sources/")
```

Comment:
nit: maybe `contentsEqual` instead of pass.

Reply:
Fixed.